### PR TITLE
[SER-254] Seed Admin Username Joi Support Email

### PR DESCRIPTION
**Pull request by mountHouli**
#### What's this PR do?

Previously, Joi validation required that `AUTH_SERVICE_SEED_ADMIN_EMAIL` be an email address but `AUTH_SERVICE_SEED_ADMIN_USERNAME` be alphanumeric.

We want to be able to set both to the same value. Therefore, Joi validation for `AUTH_SERVICE_SEED_ADMIN_USERNAME` was changed to an email address.

#### Related JIRA tickets:

https://jira.amida.com/browse/SER-254

#### How should this be manually tested?

Set `AUTH_SERVICE_SEED_ADMIN_USERNAME` to something that isn't an email address and isn't a plain alphanumeric, such as `a.b.c`. Joi validation should fail.

Set it to an email address or to an alphanumeric value. Joi validation should pass.

#### Any background context you want to provide?

On all deployments of Orange, it was continually frustrating to manage the `_USERNAME` and `_EMAIL` being different values.

- [x] No linter errors
- [x] Passes all tests

### CODE REVIEW

File: src/config/config.js [hunk 1]
```diff
@@ -44,21 +44,19 @@ const envVarsSchema = Joi.object({
...
     AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.string()
-        .alphanum()
-        .min(3)
-        .max(30)
+        .email({ minDomainAtoms: 2 })
         .required()
-        .default('admin')
-        .description('Admin username for seeding only'),
+        .default('admin@example.com')
+        .description('User name of the Seed Admin user. Used in the for seeding process only.'),
```
Comment (orndorffgrant) [inline src/config/config.js]:
```suggestion
        .description('User name of the Seed Admin user. Used in the seeding process only.'),
```

Comment (mountHouli) [reply src/config/config.js]: Whoah! The ability to commit suggestions directly. Awesome!

**Comment by mountHouli** [comment]
Thanks for the comments! I think forcing them to be the same will save a headache later.

**Comment by mountHouli** [comment]
Used `Joi.alternatives()` to support email and alphanum. https://github.com/amida-tech/amida-auth-microservice/pull/94/commits/1cca2e7e916c0f6604cd002e6f1349f5cc2bf3a8

File: src/config/config.js [hunk 2]
```diff
@@ -43,22 +43,28 @@ const envVarsSchema = Joi.object({
...
     AUTH_SERVICE_REFRESH_TOKEN_MULTIPLE_DEVICES: Joi.bool()
         .default(false),
-    AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.string()
-        .alphanum()
-        .min(3)
-        .max(30)
-        .required()
-        .default('admin')
-        .description('Admin username for seeding only'),
+    AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.alternatives([
```
Comment (rmharrison) [inline src/config/config.js]: `alternatives`!

I knew `Joi` had to support multiple validations, but didn't see how in a skim of the docs.

**Pull request merged**